src/rookify/modules/analyze_ceph/main.py

```diff
@@ -3,11 +3,14 @@
 import json
 
 from ..module import ModuleHandler
+from rookify.logger import getLogger
 
 class AnalyzeCephHandler(ModuleHandler):
 
     def run(self) -> dict:
 
+        log = getLogger()
```

Comment:
Don't instanciate logger in the actual module but in ModuleHandler class

Reply:
ok i instantiated it in the module.py